Makefile

```diff
@@ -32,3 +49,9 @@ run-link-checker:
 
 check-links: clean production-build link-checker-setup run-link-checker
 
+# Adding additional link checks based on https://github.com/grpc/grpc.io/blob/main/Makefile
+check-internal-links: production-build link-checker-setup run-link-checker
+
+check-all-links: production-build link-checker-setup
+	bin/htmltest --conf .htmltest.external.yml
+
```

Comment:
Btw, I haven't yet vetted link checking on grpc.io. I'll do that soon and get back to you. In the meantime, this can go in now.